frontend/src/Guest/RegisterForm.tsx

```diff
@@ -21,16 +22,25 @@ export default function RegisterForm() {
     const [email, setEmail] = useState("");
     const [password, setPassword] = useState("");
     const [confirmPassword, setConfirmPassword] = useState("");
+    const [error, setError] = useState("");
 
     const handleFormSubmit = (event: ChangeEvent<HTMLFormElement>) => {
         event.preventDefault();
+        if (!isValidEmail(email)) {
+            setError("Email is invalid");
+        } else {
+            setError("");
+        }
```

Comment:
Do you still want to continue sending the form and the registration process even the email is invalid? 🤔